### PR TITLE
Updated the code to Julia 1.1

### PR DESCRIPTION
**Pull request by Tilstandsrommet**
Shifted peaks and valleys to julias 1-indexing
Added R value to the calculation
Added readability of count_cycles()
shift!() has new name popfirst!()
linspace() has new name range()

### CODE REVIEW

File: src/Rainflow.jl [hunk 1]
```diff
@@ -3,8 +3,8 @@ import Base.show
 
 export sort_peaks, find_boundary_vals, count_cycles, sum_cycles
 
-""" This function sort out points where the slope is changing sign."""
```
Comment (dhoegh) [inline src/Rainflow.jl:6]: As general comment please do not remove the triple quotes. They yields the sentence as help, see picture below
image

File: src/Rainflow.jl [hunk 2]
```diff
@@ -71,23 +69,27 @@ mutable struct Cycles_bounds #
...
+        cycle.Rvalue < bounds.min_R && setfield!(bounds, :min_R, cycle.Rvalue)
+        cycle.Rvalue > bounds.max_R && setfield!(bounds, :max_R, cycle.Rvalue)
     end
     return bounds
 end
 
-""" Returns the range index the value is belonging in """
-function find_range{T<:Real}(interval::Array{T,1},value)
+# Returns the range index the value is belonging in
+function find_range(interval,value)
```
Comment (dhoegh) [inline src/Rainflow.jl]: Please keep the function as shown below, as the function assumes the interval is a vector
`function find_range(interval::Array{T,1},value) where {T <: Real}`

**Review by dhoegh — COMMENTED** [review]
If you address my comments i will merge your changes.

**Comment by Tilstandsrommet** [comment]
I fixed what you said!

**Pull request merged**